lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -213,7 +227,20 @@ window.loadSelectedAggregations = function() {
 };
 
 window.listenForStatus = function(callback) {
-  log.events.addListener('status', callback);
+  log.events.addListener('status', function(args) {
```

Comment:
nit: maybe just `log` or something other than `args`? staring at it a bit assuming it was a list ala `arguments` and not just the one input